Duplicati/GUI/Duplicati.GUI.TrayIcon/Program.cs

```diff
@@ -114,10 +114,9 @@ public static int Main(string[] _args)
                 // Tell the hosted server it was started by the TrayIcon
                 Server.Program.Origin = "Tray icon";
 
-                var cert = Server.Program.DataConnection.ApplicationSettings.ServerSSLCertificate;
                 var scheme = "http";
 
-                if (cert != null && cert.HasPrivateKey)
+                if (!string.IsNullOrEmpty(databaseConnection.ApplicationSettings.ServerSSLCertificate))

```

Comment:
Would this fail with `--webservice-usehttps=false` ?

Reply:
again for backward compatibility with users that have now certificate set, but dont use new ServerUseHTTPS parameter

Reply:
without that they would fall back to http with new version.
its true, that without backward compatibility, it would simplify code a little bit

Reply:
I see, I was not reading it with a good understanding of upgrade for existing certificate users.